File: Java/src/veterinaria/Cliente.java
package veterinaria;

import java.util.ArrayList;

public class Cliente {
    private String nombre;
    private ArrayList<Mascota> mascotas = new ArrayList<Mascota>();
    
    public void agregarMascota(Mascota mascota) {
        this.mascotas.add(mascota);
    }
    
    public String reporteMascotas() {
        String mensaje = "";
        
        for (Mascota mascota : this.mascotas) {
            mensaje += mascota.presentarse();
            mensaje += "\n";
        }
        
        return mensaje;
    }
}

File: Python/Veterinaria/Cliente.py
class Cliente(object):
    def __init__(self, nombre):
        self.nombre = nombre
        self.mascotas = []

    def agregarMascota(self, mascota):
        self.mascotas.append(mascota)

    def reporteMascotas(self):
        mensaje = ""
        for mascota in self.mascotas:
            mensaje += mascota.presentarse()
            mensaje += "\n"
        return mensaje

